paddle/fluid/operators/nccl_op_test.cu.cc

```diff
@@ -236,48 +236,49 @@ TEST_F(NCCLTester, ncclReduceOp) {
 }
 
 // ncclBcastOp with desc
```

Comment:
Should we have an issue to remind us about this temporary disabling?

And how about adding a comment here mentioning the issue?

Reply:
Yes, it's here: https://github.com/PaddlePaddle/Paddle/issues/9540 , will add it to comment.

Reply:
Done.